javascript/src/main/pages/Home/Home.js

```diff
@@ -1,21 +1,36 @@
 import React from "react";
+import { useState } from "react";
 import { Jumbotron } from "react-bootstrap";
-import {useAuth0} from "@auth0/auth0-react";
+import { useAuth0 } from "@auth0/auth0-react";
 import { Redirect } from "react-router-dom";
+import BasicCourseSearchForm from "../../components/BasicCourseSearch/BasicCourseSearchForm";
+import JSONPrettyCard from "../../components/Utilities/JSONPrettyCard";
 
 const Home = () => {
-    const { isAuthenticated } = useAuth0();
+    
+    // every function that starts with "use" is a hook
+    // e.g. useState, useSWR, useAuth0
 
+    const { isAuthenticated } = useAuth0(); // a hook
```

Comment:
This is unused; consider removing auth0 from this component entirely for the moment?